AUTHORS

```diff
@@ -0,0 +1 @@
+Contributors to Disq:
```

Comment:
We should add to the contributing guidelines how the name "Disq" should be written in files mentioning it: so disallow "DISQ", "distq" or depending on the file using it (java class, etc.).